hack/arktos-up.sh

```diff
@@ -554,6 +559,12 @@ if [[ "${START_MODE}" != "nokubelet" ]]; then
     esac
 fi
 
+# Applying mizar cni
+if [[ "${CNIPLUGIN}" = "mizar" ]]; then
```

Comment:
For consistency, use '==' . I think this is not the right usage.

Reply:
Thanks @vinaykul , we will modify this and test it again.